src/biome/text/pipeline.py

```diff
@@ -215,6 +215,8 @@ class Pipeline:
     __LOGGER = logging.getLogger(__name__)
     __TRAINING_CACHE_DATA = "instances_data"
 
+    logging.getLogger("allennlp").setLevel(logging.WARNING)
+
```

Comment:
Does this disable the `verbose` option of the `train` command?

Reply:
You're right. I'll make the changes to preserve the `verbose` feat.

Reply:
Reverted